src/app/services/assets.service.ts

```diff
@@ -11,8 +11,7 @@ export class AssetsService {
 
   get(asset: any): string {
     if ( asset === null ) {
-      // @TODO insert default asset for null asset retrieval
-      return '';
+      return `${environment.apiBase}/assets/default_profile.png`;
```

Comment:
Since this is the general assets service, I would like a more general name; `default.png` I think works nicely